procurement_purchase_no_grouping/models/product_category.py

```diff
@@ -10,15 +10,18 @@ class ProductCategory(models.Model):
 
     procured_purchase_grouping = fields.Selection(
         [
+            ("system", "System default"),
             ("standard", "Standard grouping"),
             ("line", "No line grouping"),
             ("order", "No order grouping"),
         ],
         string="Procured purchase grouping",
-        default="standard",
+        required=True,
```

Comment:
Please don't change this: not putting anything is for using the upper configuration. The criteria order should be: check the product, if no policy, check the category, and if not, go to system parameter.

Reply:
@pedrobaeza not sure I follow this one. My findings looking into database is that when you install the module it will fill in the default value to all product categories. You can then change it to None but that kind of defeats my use case (which might be specific I admit). So if None is used to indicate to go one level higher then I guess there should be no default value. Or am I missing something?

Reply:
Yes, both things: I don't want any default value nor being required.

Reply:
> Yes, both things: I don't want any default value nor being required.

OK, that makes sense. So None will fall back to system default. Will do right away.